clean.py

```diff
@@ -35,32 +35,40 @@ def revert_bool_output(examples, filter_function):
     booleans = filter_function(examples)
     return [not boolean for boolean in booleans]
 
+def filter_diff_text(examples, in_text_col, out_text_col):
+    return [text_in!=text_out for text_in, text_out in zip(examples[in_text_col], examples[out_text_col])]
+    
 def get_args():
     parser = argparse.ArgumentParser()
     parser.add_argument("--dataset-path", type=str, required=True)
     parser.add_argument("--maps-and-filters", nargs="*", type=str, required=True)
     parser.add_argument("--save-path", type=str, required=True)
-    parser.add_argument("--checks-save-path", type=str, default=None)
+    parser.add_argument("--checks-save-path", type=Path, default=None)
     parser.add_argument("--num-proc", type=int, default=1)
     parser.add_argument("--batch-size", type=int)
     parser.add_argument("--load-arrow-file", action="store_true")
+    parser.add_argument("--sampling-size-map-checks", type=int, default=None)
+    parser.add_argument("--sampling-size-filter-checks", type=int, default=None)
```

Comment:
@lvwerra , As you rightly pointed out in your comment I added these 2 new arguments to select the number of lines to take into account for the checks. Let me know if the result if fine for you :smile: